src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
```

Comment:
The interface name 'AreaCalculator' is not recommended to use verbs. A better name would be something that represents an entity or a concept, such as 'CalculableArea' or 'AreaComputable'.